bigbluebutton-tests/puppeteer/stress.obj.js

```diff
@@ -0,0 +1,25 @@
+const { toMatchImageSnapshot } = require('jest-image-snapshot');
```

Comment:
This is about importing the visual regression libs, while it's not used here, so it should be removed.